.circleci/config.yml

```diff
@@ -0,0 +1,81 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/go/src/github.com/docker/libnetwork
+  docker:
+    - image: 'circleci/golang:1.9.6'
+      environment:
+          CODEPATH: "~/go/src/github.com/docker/libnetwork"
+
+jobs:
+  builder:
+    <<: *defaults
+    steps:
+      - checkout
+      - setup_remote_docker:
+          reusable: true
+          exclusive: false
+      - run: make circle-ci
+      #- run: make build-builder
+      # - run: mkdir -p images
```

Comment:
I'm a bit confused here w.r.t. the build-builder target.  It's added to the Makefile below, but it seems to be commented out here.  Is it needed or not?